internal/test/wmcb/wmcb_test.go

```diff
@@ -199,6 +193,23 @@ func (vm *wmcbVM) initializeTestBootstrapperFiles() error {
 		return fmt.Errorf("unable to create remote directory %s: %v", remoteDir, err)
 	}
 
+	kubeNodeURL, err := framework.GetKubeNodeURL()
```

Comment:
I thinkw we can wrap getting `kubeNodeURL, KubeNodeSHA` into a single method and get the package as return value for that method.

Reply:
I agree just return `pkgInfo`